### DISCUSSION TITLE
Facade class not found

### DISCUSSION THREAD

**Discussion by larswoltersdev** [Jeroen-G/laravel-packager#160] [Q&A]
I added the package to a fresh Laravel project (non-Sail). 

Then I did `php artisan packager:new testvendor testpackage`

My `Testpackage` class contains:

```php
<?php

namespace Testvendor\Testpackage;

class Testpackage
{
    public function test(): string
    {
        return 'test';
    }
}
```

Now, when I try to use the facade class in web.php, I get `Class "Testvendor\Testpackage\Facades\Testpackage" not found`.

```php
Route::get('/test', function () {
    dd(TestPackage::test());
});
```

I changed nothing else. What is happening?

**Comment by Jeroen-G** [comment]
Can I see the composer.json of your package and app?
